osu.Game/Screens/Select/FilterQueryParser.cs

```diff
@@ -66,6 +66,10 @@ private static bool tryParseKeywordCriteria(FilterCriteria criteria, string key,
                 case "lastplayed":
                     return tryUpdateDateAgoRange(ref criteria.LastPlayed, op, value);
 
+                case "unplayed":
+                    criteria.LastPlayed.Max = DateTimeOffset.MinValue.AddDays(1);
```

Comment:
The add one days is just to catch any weirdness? Did you find a specific case? Realm inaccuracy?

Reply:
`DateTimeOffset.MinValue` is the filtered value:

https://github.com/ppy/osu/blob/a077acc599596fbbfe39ab517961463b34c96310/osu.Game/Screens/Select/Carousel/CarouselBeatmap.cs#L68

An alternative could be...
```
criteria.LastPlayed.Min = DateTimeOffset.MinValue;
criteria.LastPlayed.Max = DateTimeOffset.MinValue;
criteria.LastPlayed.IsLowerInclusive = true;
criteria.LastPlayed.IsUpperInclusive = true;
```
If that makes it better.

Reply:
Right.

I *think* I prefer that alternative as it's more explicit to what's going on.